src/components/util/checkbox-list.js

```diff
@@ -53,7 +52,7 @@ const CheckboxList = ({
             const newChecked = new Set(checked);
             let valueToDelete = undefined;
```

Comment:
"Functional style" brings more conciseness and const :
`const valueToDelete = newChecked.find((e) => e.uuid === value?.uuid);`